src/core/grpc_service_v2.proto

```diff
@@ -778,6 +788,59 @@ message ModelInferResponse
   repeated InferOutputTensor outputs = 4;
 }
 
+//@@
+//@@.. cpp:var:: message ModelStreamInferResponse
+//@@
+//@@   Response message for ModelStreamInfer.
+//@@
+message ModelStreamInferResponse
+{
+  //@@
+  //@@  .. cpp:var:: message Status
+  //@@
+  //@@     The status of the request.
+  //@@
+  message Status
+  {
+    //@@
+    //@@    .. cpp:var:: int32 code
+    //@@
+    //@@       The status code
+    //@@
+    int32 code = 1;
+
+    //@@
+    //@@    .. cpp:var:: string message
+    //@@
+    //@@       The message describing the error.
+    //@@
+    string message = 2;
+  }
+
+  //@@
+  //@@  .. cpp:var:: Status status
+  //@@
+  //@@     Holds the status of the request.
+  //@@
+  Status status = 1;
+
+  //@@
+  //@@  .. cpp:var:: ModelInferResponse infer_response
+  //@@
+  //@@     Holds the results of the request.
+  //@@
+  ModelInferResponse infer_response = 2;
+
+  //@@
```

Comment:
sequence_id is independent of streaming so I don't think we want it here. I think it should be in the response parameters. Let me look at the sequence extension doc and see...